projet/projects/urls.py

```diff
@@ -11,4 +11,20 @@
         views.ProjectDetailUpdateDelete.as_view(),
         name="update-delete",
     ),
+    path(
+        "accept/<int:project_id>/<int:contributor_id>/",
+        views.AcceptContributor.as_view(),
+        name="accept-contribution",
+    ),
```

Comment:
At first for this API route i only added the contribution id and used permissions . But i realized that may cause issues, so i did it in a way that the project id will also have to be sent along with the contribution id.